Editor/CompileCesiumForUnityNative.cs

```diff
@@ -255,6 +277,17 @@ public static LibraryToBuild GetLibraryToBuild(PlatformToBuild platform, string
             if (platform.platformGroup == BuildTargetGroup.Android)
                 library.Toolchain = "extern/android-toolchain.cmake";
 
+            if (platform.platformGroup == BuildTargetGroup.iOS){
+                var outputPath = Path.Combine(library.SourceDirectory, "../Plugins/iOS");
+            if (Directory.Exists(outputPath))
+                Directory.Delete(outputPath, true);
+            Directory.CreateDirectory(outputPath);
```

Comment:
Are these lines needed, or were they just for testing? If they're needed, fix the indentation and maybe add a comment explaining why.

Reply:
Figured out that these lines were redundant.